src/vorta/views/archive_tab.py

```diff
@@ -248,6 +259,9 @@ def populate_from_profile(self):
             self.mount_points, repo_mount_points = get_mount_points(profile.repo.url)
             if repo_mount_points:
                 self.repo_mount_point = repo_mount_points[0]
+                mount_point_filename = os.path.basename(self.repo_mount_point)
+                if mount_point_filename.startswith("vorta-quick-mount-"):
```

Comment:
Assuming this should be safe right?  The other option is to delete the quick mount folder when vorta exits.

Reply:
I'd consider this safe.